packages/react-components/src/components/ParticipantItem.tsx

```diff
@@ -180,7 +181,7 @@ export const ParticipantItem = (props: ParticipantItemProps): JSX.Element => {
       ref={containerRef}
       role={'menuitem'}
       data-is-focusable={true}
-      className={mergeStyles(participantItemContainerStyle(me), styles?.root)}
+      className={mergeStyles(participantItemContainerStyle(me, menuItems ? true : false), styles?.root)}
```

Comment:
can be simply `!!menuItem`

Also, I generally advice against flag arguments: https://moderatemisbehaviour.github.io/clean-code-smells-and-heuristics/functions/f3-flag-arguments.html?msclkid=1506d94ac83c11ecb82a9b5b09dc259b

Consider creating two different functions, `participantItemContainerStyleNonClickable` and `participantItemContainerStyleClickable`.

OR

[my preferred], keep the flag argument, but make the arguments easier to understand at the callsite by using an options bag: 
```
participantItemContainerStyle({localParticipant: me, clickable: !!menuItems))

const participantItemContainerStyle = (options: {localparticipant: boolean, clickable: boolean}) => {
...
}
```

Reply:
udpated